tests/src/lib.rs

```diff
@@ -211,6 +225,25 @@ mod tests {
         roundtrip::<foo::bar_baz::FooBarBaz>(&buf).unwrap();
     }
 
+    #[test]
+    fn test_custom_type_attributes() {
+        // We abuse the ident conversion protobuf for the custom attribute additions. We placed
+        // `Ord` on the FooBarBaz (which is not implemented by ordinary messages).
+        let mut set1 = BTreeSet::new();
+        let msg1 = foo::bar_baz::FooBarBaz::default();
+        set1.insert(msg1);
+        // Similar, but for oneof fields
+        let mut set2 = BTreeSet::new();
+        let msg2 = oneof_attributes::Msg::default();
+        set2.insert(msg2.field);
+    }
+
+    #[test]
+    fn test_custom_field_attributes() {
```

Comment:
Is this test case getting us more coverage than the #[deny(missing_docs)] trick?

Reply:
Not really. I just never learned to kill tests that are no longer needed unless they fail. It is gone now, because it was so ugly anyway.